packages/contracts/test/plugins/token/distribution/merkle-distributor.ts

```diff
@@ -71,20 +75,32 @@ describe('MerkleDistributor', function () {
     });
   });
 
-  describe('Upgrade', () => {
-    beforeEach(async function () {
-      this.upgrade = {
-        contract: distributor,
-        dao: dao,
-        user: signers[8],
-      };
-      await distributor.initialize(dao.address, token.address, ZERO_BYTES32);
+  describe('Upgrades', () => {
+    let legacyContractFactory: ContractFactory;
+    let currentContractFactory: ContractFactory;
+
+    before(() => {
+      currentContractFactory = new MerkleDistributor__factory(signers[0]);
     });
 
-    shouldUpgradeCorrectly(
-      UPGRADE_PERMISSIONS.UPGRADE_PLUGIN_PERMISSION_ID,
-      'DaoUnauthorized'
-    );
+    it('from v1.0.0', async () => {
+      legacyContractFactory = new MerkleDistributor_V1_0_0__factory(signers[0]);
```

Comment:
perhpas we can add a comment that we didn't upgrade this contract in this version, and it is just a precaustion to make sure we didn't accedentially changed anything